ansibulled/changelog/changelog_generator.py

```diff
@@ -144,8 +144,8 @@ def _collect(self, squash: bool = False, after_version: Optional[str] = None,
                 for section, lines in fragment.content.items():
                     if section == self.config.prelude_name:
                         if entry_fragment:
-                            LOGGER.info('skipping prelude in version %s due to newer '
```

Comment:
Pylint explicitly mentioned to use `%`. It's good to know it also allows `{}`!

Reply:
I think it's a toggle in the config file.  Since i changed the config file, i figured i should fix this as well :-)